packages/arb-token-bridge-ui/src/hooks/CCTP/fetchCCTPLimits.ts

```diff
@@ -0,0 +1,39 @@
+import { readContract } from '@wagmi/core'
+import { fiatTokenProxyAbi, tokenMinterAbi } from '../../util/cctp/abi'
+import { getContracts, CCTPSupportedChainId } from './useCCTP'
+
+// See https://developers.circle.com/stablecoin/docs/cctp-technical-reference#transaction-limits
+// Note: fetchMinterAllowance returns allowance on the DESTINATION chain
```

Comment:
use /** */ to make IDE show jsdoc comment when hovering over the method `fetchMinterAllowance` at usage